cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -139,29 +158,120 @@ var _ = Describe("[sriov] NUMA node alignment", Ordered, func() {
 			g.Expect(actualPod.Status.QOSClass).To(Equal(corev1.PodQOSGuaranteed))
 		}, 30*time.Second, 1*time.Second).Should(Succeed())
 
+		By("Validate Pod NUMA Node")
+		expectPodCPUsAreOnNUMANode(pod, 1)
```

Comment:
I think you need to add this one to the others tests no?
@zeeke WDYT?

Reply:
It was added to all testts that didn't fail to deploy a pod.